releasenotes/notes/grpc-stats.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - 43908
```

Comment:
Is https://github.com/istio/istio/issues/44144 another reference or a better reference?